### DISCUSSION TITLE
Any update on svelte support?

### DISCUSSION THREAD

**Discussion by ghost** [rome/tools#4183] [Suggestions]
I am considering utilising Rome for a new project and am wondering if Svelte Support is possible, I haven't been able to find anything in searching online.

Any help would be greatly appreciated.

**Comment by GabenGar** [comment]
Does svelte use non-standard syntax? It works for react because typescript supports react syntax. So the answer depends on if typescript supports svelte.

**Reply by ghost** [reply]
Pretty sure Svelte has Typescript support... Unsure if TypeScript has Svelte support.

**Comment by thelinuxlich** [comment]
I'd love that feature too
